app/angular/src/client/preview/angular/ComponentClassFromStoryComponent.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Inject,
+  OnDestroy,
+  Type,
+  ViewChild,
+  ViewContainerRef,
+} from '@angular/core';
+import { Subscription, Subject } from 'rxjs';
+import { map, skip } from 'rxjs/operators';
+
+import { ICollection } from '../types';
+import { STORY_PROPS } from './app.token';
+import {
+  ComponentInputsOutputs,
+  getComponentDecoratorMetadata,
+  getComponentInputsOutputs,
+} from './NgComponentAnalyzer';
+import { RenderNgAppService } from './RenderNgAppService';
+
+const distinctComponentInputsOutputsWithProps = (
```

Comment:
If I understand correctly the function I would suggest renaming it to something like `getNamesOfInputsOutputsDefinedInProps`